src/custom-element-widget.js

```diff
@@ -110,14 +110,18 @@ function makeInit(tagName, definitionFn) {
     let widget = this;
     let element = createContainerElement(tagName, widget.properties);
     let propertiesProxy = makePropertiesProxy();
-    let domUI = applyToDOM(element, definitionFn, propertiesProxy);
+    let domUI = applyToDOM(
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).